internal/common/eventutil/eventutil.go

```diff
@@ -864,6 +864,24 @@ func EventSequenceFromApiEvent(msg *api.EventMessage) (sequence *armadaevents.Ev
 				JobRunSucceeded: &armadaevents.JobRunSucceeded{
 					RunId: runId,
 					JobId: jobId,
+					ResourceInfos: []*armadaevents.KubernetesResourceInfo{
+						{
+							ObjectMeta: &armadaevents.ObjectMeta{
+								Namespace:    m.Succeeded.NodeName,
```

Comment:
NameSpace != NodeName

Reply:
ah good spot.  I actually copied this from the equivalent `JobRunRunning` function and sure enough we have the same issue there- I've fixed both